### ISSUE TITLE
Implement new subdomain - fees calculation 

### ISSUE THREAD

**Issue by pilloPl** [pilloPl/library#6]
1. Start with event storming

2. Start with new package in current implementation

3. Update arch diagrams

4. Minimal model-code gap.

**Comment by dosdebug** [comment]
I want to be part of this whole scenario too. I would like to be part of every step mentioned in this too. @bslota @pilloPl

**Comment by bslota** [comment]
Good to hear, @dosdebug. We'll keep you in the loop as soon as we're ready to start.

**Comment by gruzdrivendev** [comment]
If it's not a problem, I'd also like to participate.

**Comment by dosdebug** [comment]
@bslota Any update when are we going to start on this?

**Comment by bslota** [comment]
@dosdebug Not before the beginning of May (Easter, holidays, you know ;) ) I promise not to start it without you :)

**Comment by dosdebug** [comment]
Thanks a lot @bslota

**Comment by ArturWierzbicki** [comment]
I'd also love to take part in this!
@bslota @pilloPl

**Comment by pilloPl** [comment]
I think we will start the work next month :) let's have a hangout session before

**Comment by dosdebug** [comment]
@pilloPl Perfect

**Comment by krzykrucz** [comment]
If that's possible and not too many people interested already, I'd be happy to contribute in any way 😃

**Comment by pilloPl** [comment]
The more people, the better :) We will event storm this example, probably @bslota will lead this storming session since I am short with time lately. 

After the storming, we will propose some model - probably in offline mode (I will try to jump at this stage).

Then we can divide work into implementation details.

@bslota, can you please schedule a remote storming session (since it's probably impossible to meet :) ?

**Comment by bslota** [comment]
Yep, I will schedule a session to take place beginning of June. Will all let you know, putting a link to hangout here in the discussion.

**Comment by bslota** [comment]
Hi All! Please find [the link to doddle poll](https://doodle.com/poll/bywhbqa8idhhddgc). I proposed a couple of time slots for our session. The form should adjust time zone according to your location. Please let me know if these hours does not work for you. @dosdebug @gruzdrivendev @ArturWierzbicki @pilloPl

**Comment by thomdabrowski** [comment]
If it's not too late, I would like to help as well!

**Comment by tulliocba** [comment]
I would like to participate as a listener, because my English is not very good.

**Comment by bslota** [comment]
I will let you know about the final time of the workshop the beginning of next week

**Comment by pilloPl** [comment]
@michal-michaluk will join you! He will be co-facilitating or playing domain expert

**Comment by bslota** [comment]
Great to hear! @michal-michaluk could you please share your time preferences in doodle poll?

**Comment by odedia** [comment]
@bslota thanks for the doodle link. Seems like it would start at 5am my time and around 3am London time, so would be tough for people from Europe to join. Any chance you'll be able to record the session (hangouts/zoom)?
Thanks.

**Comment by bslota** [comment]
@odedia thanks for spotting it! I must have made a mistake while updating the doodle. It should be PM everywhere. Sorry for that! Everyone who already voted - please share your preferences again. Apologies for that!

**Comment by adamkulawik** [comment]
@bslota @pilloPl I know it's quite late, but I would also like to contribute to that issue. Is that still possible?

**Comment by bslota** [comment]
@adamkulawik sure it is! Please make your vote in the poll.

**Comment by knizamov** [comment]
Could I also please participate as a listener in the Event Storming session?

**Comment by onlyahobo** [comment]
@bslota, I would like to participapte by your permission.

**Comment by karthik-cbe** [comment]
@bslota  please include me in your discussion.

**Comment by jaroslawu** [comment]
If you don't mind, I'd like to join in as well.

**Comment by ucinskij** [comment]
If you don't mind, I'd like to join in as well.

**Comment by bslota** [comment]
Thank you everyone for participating in the poll. Trying to fit the majority of you, we will be holding this session on July 9th (Tuesday) at 4:00 PM. Here is [the link](https://hangouts.google.com/call/VWqc2uHV9kQzz29M_eZLAEEI?no_rd) to hangouts meeting.

**Comment by bslota** [comment]
Big thanks to everyone who attended the last session! If you want to continue with Design Level tactical session with us, feel free to fill the [Doodle poll](https://doodle.com/poll/4vf4pexexfdhc9in) and declare best time for you.

**Comment by aleksander-siewierski** [comment]
@bslota is time of second workshop already scheduled? On Doodle no final option choosed.

**Comment by bslota** [comment]
Design Level EventStorming will be handled on 1st of August from 4PM till 7PM. Please find [the link to video hangouts](https://hangouts.google.com/call/WqfiqsuAffUPGNfB8RkNAEEI?no_rd).

**Comment by aleksander-siewierski** [comment]
@bslota I cannot join :( could this meeting be recorded?
